runhouse/resources/hardware/cluster.py

```diff
@@ -895,6 +895,7 @@ def restart_server(
                 else ""
             )
             + " --from-python"
+            + f" --log-level {logging.getLevelName(logging.root.level)}"
```

Comment:
re: whether to have this API support the same level of control as the CLI, maybe we just add a `log_level` arg here and pass that in instead? pretty harmless to add one extra param